examples/tests/valid/api_path_vars.w

```diff
@@ -0,0 +1,32 @@
+bring cloud;
+
+let api = new cloud.Api();
+
+resource Fetch {
+  extern "./api_path_vars.js" inflight get(url:str): Json;
+  init(){}
+}
```

Comment:
Let's try to keep our Wing code well formatted to set a good example for the community:

```suggestion
resource Fetch {
  extern "./api_path_vars.js" inflight get(url: str): Json;
  init() { }
}
```